python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -824,7 +839,16 @@ def _clone_var(self, block, var, persistable=True):
             persistable=persistable)
 
     def _append_split_op(self, program, gradblocks):
-        # Split variables that need to be split and append respective ops
+        """
+        Split variables that need to be split and append respective ops
```

Comment:
Split variables with the size larger than the minimum size, and append split_byref_op and concat_op for them.